pkg/destroy/powervs/powervs.go

```diff
@@ -718,7 +719,7 @@ func (o *ClusterUninstaller) ServiceInstanceNameToGUID(ctx context.Context, name
 }
 
 func (o *ClusterUninstaller) contextWithTimeout() (context.Context, context.CancelFunc) {
```

Comment:
As it is now, this has no dep on the `ClusterUninstaller` so you could just change it to:
```suggestion
func contextWithDefaultTimeout() (context.Context, context.CancelFunc) {
```

Reply:
updated